NeoML/src/TraditionalML/GradientBoost.cpp

```diff
@@ -667,9 +699,19 @@ void CGradientBoost::buildFullPredictions( const IMultivariateRegressionProblem&
 				CSparseFloatVectorDesc vector;
 				matrix.GetRow( index, vector );
 
-				for( int j = 0; j < models.Size(); j++ ) {
-					predictCache[j][index].Value += CGradientBoostModel::PredictRaw( models[j], predictCache[j][index].Step,
-						params.LearningRate, vector );
+				CFloatVector predictions( problem.GetValueSize(), 0.0 );
```

Comment:
Don't use CFloatVector here! CFastArray<> will be better! You can return CFastArray<> as a parameter from CGradientBoostModel::MultivariatePredictRaw.